packages/quantic/force-app/main/default/lwc/quanticInsightInterface/quanticInsightInterface.js

```diff
@@ -93,4 +111,39 @@ export default class QuanticInsightInterface extends LightningElement {
   get input() {
     return this.template.querySelector('input');
   }
+
+  bindAriaLiveEvents() {
+    this.template.addEventListener(
+      'arialivemessage',
+      this.handleAriaLiveMessage.bind(this)
+    );
+    this.template.addEventListener(
+      'registerregion',
+      this.handleRegisterAriaLiveRegion.bind(this)
+    );
+    this.ariaLiveEventsBound = true;
+  }
+
+  handleAriaLiveMessage(event) {
+    /** @type {import('quanticAriaLive/quanticAriaLive').IQuanticAriaLive} */
+    const ariaLiveRegion = this.querySelector('c-quantic-aria-live');
+    if (ariaLiveRegion) {
+      ariaLiveRegion.updateMessage(
+        event.detail.regionName,
+        event.detail.message,
+        event.detail.assertive
```

Comment:
question: Is loading considered critical here? I believe assertive should be used as little as possible as it can be annoying. Have you considered `polite` instead here? 

From the [ARIA docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Live_Regions#:~:text=aria%2Dlive%3D%22assertive,be%20used%20sparingly.)

Reply:
We are not using assertive aria live regions here.

Here we are only calling the method `updateMessage` of the QuanticAriaLive component and we are passing to it, as parameters, the the payload of the sent event.

If we take a step back and follow the creation of the event we see that in the payload of it the value of `assertive` is set to `false`. this would make the QuanticArialive component create a polite aria live region and not an assertive region.